libs/application/templates/general-fishing-license/src/fields/Overview/index.tsx

```diff
@@ -115,26 +122,84 @@ export const Overview: FC<FieldBaseProps> = ({ application, goToScreen }) => {
           )}
         </GridRow>
       </ReviewGroup>
-
-      <ReviewGroup isLast>
+      <ReviewGroup>
         <Box
           border="standard"
           borderRadius="large"
           padding={3}
           width="full"
           display="flex"
           justifyContent="spaceBetween"
-          marginBottom={5}
+          marginBottom={2}
         >
           <ShipInformation ship={ship} seaworthinessHasColor />
         </Box>
+      </ReviewGroup>
+      <ReviewGroup
+        isLast
+        editAction={() => changeScreens('fishingLicenseFurtherInformation')}
+      >
         <GridRow>
-          <GridColumn span={['9/12', '9/12', '9/12', '5/12']}>
+          <GridColumn span={['9/12', '9/12', '9/12', '5/12']} paddingBottom={3}>
```

Comment:
Seeing as these 5 overview items are the same code we could get the answers and render each with a function which accepts the label and value which would end up in a lot less code.